packages/pwa-kit-dev/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ## v3.6.0-dev (Apr 17, 2024)
+- Add logger logger to print logs generated by PWA Kit packages [#1822](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1822)
```

Comment:
Oh yeah, updating the changelog files... do we need to update the ones for other packages too that got affected in this PR?

Reply:
I think we only need  to add the entry to the package where we are adding the PWAKitLogger class.